src/main/java/com/example/demo/model/Match.java

```diff
@@ -0,0 +1,62 @@
+package com.example.demo.model;
+
+import lombok.AccessLevel;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Data
+@Entity
+@NoArgsConstructor
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@Table(name = "MATCHES")
+public class Match {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    int id;
+
+    @Column(nullable = false)
+    int round;
+
+    @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
1) EAGER is already default value (javax.persistence.ManyToOne#fetch)
2) Must be LAZY